build.gradle.kts

```diff
@@ -574,11 +574,6 @@ tasks.register("javaExamplesDataflowPrecommit") {
   dependsOn(":runners:google-cloud-dataflow-java:examples-streaming:preCommit")
 }
 
-tasks.register("runBeamDependencyCheck") {
-  dependsOn(":dependencyUpdates")
```

Comment:
Looks like this comes from https://github.com/ben-manes/gradle-versions-plugin so I think we can kill https://github.com/apache/beam/blob/8ec667c8a0b746849eed6b0199e41070c49a110b/build.gradle.kts#L25

Reply:
and https://github.com/apache/beam/blob/8ec667c8a0b746849eed6b0199e41070c49a110b/ownership/JAVA_DEPENDENCY_OWNERS.yaml#L148